mono/metadata/unity-icall.c

```diff
@@ -98,16 +98,17 @@ gmt_offset(struct tm *tm, time_t t)
  *  Returns true on success and zero on failure.
  */
 guint32
-ves_icall_System_CurrentSystemTimeZone_GetTimeZoneData (guint32 year, MonoArray **data, MonoArray **names)
+ves_icall_System_CurrentSystemTimeZone_GetTimeZoneData (guint32 year, MonoArray **data, MonoArray **names, int* daylight_inverted)
```

Comment:
The managed signature uses `bool`. You should use `MonoBoolean` on native side as the parameter size is 8-bits not 32-bits

Reply:
Also, not this icall change needs synced with IL2CPP.

Reply:
per my question above, do I need a write barrier for this MonoBoolean (like we do with the lists?)

Reply:
You don't need a write barrier. That is only when storing reference types.